starfish/test/pipeline/test_gaussian_spot_detector.py

```diff
@@ -1,166 +0,0 @@
-# noinspection PyUnresolvedReferences
-from starfish.test.dataset_fixtures import (
-    synthetic_dataset_with_truth_values_and_called_spots, synthetic_single_spot_2d,
-    synthetic_two_spot_3d, synthetic_single_spot_imagestack_2d, synthetic_single_spot_imagestack_3d,
-    synthetic_two_spot_imagestack_3d, synthetic_dataset_with_truth_values, synthetic_single_spot_3d)
-
-
-def test_spots_match_coordinates_of_synthesized_spots(
-        synthetic_dataset_with_truth_values_and_called_spots):
-
-    codebook, true_intensities, image, intensities = (
-        synthetic_dataset_with_truth_values_and_called_spots)
-    assert true_intensities.shape == intensities.shape
-
-    # we do a bit of rounding, some of these may be off with 1-pixel error.
-    x_matches = np.all(np.abs(np.sort(true_intensities.coords[Features.X].values) -
-                              np.sort(intensities.coords[Features.X].values)) <= 1)
-    y_matches = np.all(np.abs(np.sort(true_intensities.coords[Features.Y].values) -
-                              np.sort(intensities.coords[Features.Y].values)) <= 1)
-    assert x_matches
-    assert y_matches
-
-
-def test_create_intensity_table(synthetic_dataset_with_truth_values_and_called_spots):
-    codebook, true_intensities, image, intensities = (
-        synthetic_dataset_with_truth_values_and_called_spots)
-
-    assert intensities.shape[0] == 5
-
-
-def test_create_intensity_table_raises_value_error_when_no_spots_detected(
-        synthetic_dataset_with_truth_values_and_called_spots):
-
-    codebook, true_intensities, image, intensities = (
-        synthetic_dataset_with_truth_values_and_called_spots)
-
-    min_sigma = 1
-    max_sigma = 10
-    num_sigma = 30
-    threshold = 1000000  # no blobs will be above this high threshold
-
-    gsd = GaussianSpotDetector(
-        min_sigma=min_sigma,
-        max_sigma=max_sigma,
-        num_sigma=num_sigma,
-        threshold=threshold,
-        blobs_stack=image,
-        measurement_type='max',
-    )
-    with pytest.raises(ValueError):
-        gsd.find(image_stack=image)
-
-
-def test_blob_log_2d(synthetic_single_spot_2d):
-    result = blob_log(synthetic_single_spot_2d, min_sigma=2, max_sigma=4, num_sigma=10, threshold=0)
-    assert np.array_equal(result, np.array([[10, 90, 2]]))
-
-
-def test_blob_log_3d(synthetic_single_spot_3d):
-    result = blob_log(synthetic_single_spot_3d, min_sigma=2, max_sigma=4, num_sigma=10, threshold=0)
-    assert np.array_equal(result, np.array([[5, 10, 90, 2]]))
-
-
-def test_blob_log_3d_flat(synthetic_single_spot_2d):
-    """
-    verify that 3d blob log works, even when the third dimension is too small to support the
-    observed standard deviation
-    """
-    data = synthetic_single_spot_2d.reshape(1, *synthetic_single_spot_2d.shape)
-    result = blob_log(data, min_sigma=2, max_sigma=4, num_sigma=10, threshold=0)
-    assert np.array_equal(result, np.array([[0, 10, 90, 2]]))
-
-
-def single_spot_detector(stack):
```

Comment:
From here down is deleted and replaced by tests that are general across the two spot detectors.